### PR TITLE
feat: Remove DOAP configuration on SystemProject and KnownUser group (DEV-4138)

### PR DESCRIPTION
**Pull request by seakayone**
- **delete SystemProject doap from test_data**
- **update documentation**
- **Remove feature to configure DAOP on SystemProject**
- **Add UpgradePlugin removing existing DOAP on the SystemProject**

### Task Description/Number

Issue Number: DEV-

### PR Type

- [ ] build/chore: maintenance tasks (no production code change)
- [ ] docs: documentation changes (no production code change)
- [x] feat: represents new features
- [ ] fix: represents bug fixes
- [ ] perf: performance improvements
- [ ] refactor: represents production code refactoring
- [ ] test: adding or refactoring tests (no production code change)
- [ ] deprecated: Deprecation warning (ideally referencing a migration guide)

### Basic requirements for bug fixes and features

- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated

### Does this PR introduce a breaking change?

- [ ] Yes

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/plugins/UpgradePluginPR3383.scala [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * Configuring default object access permissions on the SystemProject is not supported anymore.
+ * This plugin removes all default object access permissions for the SystemProject.
+ */
+class UpgradePluginPR3383 extends AbstractSparqlUpdatePlugin {
+
+  override def graphsForMigration: GraphsForMigration =
+    MigrateSpecificGraphs.from(AdminConstants.permissionsDataNamedGraph)
+
+  private val removeSystemProjectDefaultObjectAccessPermissions: ModifyQuery = {
```
Comment (seakayone) [inline webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/plugins/UpgradePluginPR3383.scala:28]: I just noticed, that this upgrade should also remove existing configurations on the `KnownUser` group.

Apart from this missing query the PR is RfR.

**Review by mpro7 — APPROVED** [review]

**Review by BalduinLandolt — APPROVED** [review]
Looks good, thanks

File: webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/plugins/UpgradePluginPR3383.scala (hunk 1 above)
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/plugins/UpgradePluginPR3383.scala:28]: => 0977eeb87aef771aed27aa9335e3a7a88d817ba0

**Comment by seakayone** [comment]
Thanks for the review @mpro7 , @BalduinLandolt . In 0977eeb87aef771aed27aa9335e3a7a88d817ba0 I have added removing possibly existing DOAPs on the KnownUser group. Please have another look at this commit.

**Review by BalduinLandolt — APPROVED** [review]
Still looks good :)

**Pull request merged**